server/server.go

```diff
@@ -39,6 +39,7 @@ import (
 	"github.com/pingcap/kvproto/pkg/pdpb"
 	"github.com/pingcap/log"
 	"github.com/pingcap/sysutil"
+	gc_server "github.com/tikv/pd/cmd/gc/server"
```

Comment:
I think server do not relay on `cmd` package, maybe move to `pkg`; see https://github.com/golang-standards/project-layout/blob/master/README_zh.md

Reply:
ok~